src/app/home/home-router.module.ts

```diff
@@ -18,7 +18,8 @@ const routes: Routes = [
       { path: 'community', component: CommunityComponent },
       { path: 'community/:nation', component: CommunityComponent },
       { path: 'resources', loadChildren: '../resources/resources.module#ResourcesModule' },
-      { path: 'meetups', loadChildren: '../meetups/meetups.module#MeetupsModule' }
+      { path: 'meetups', loadChildren: '../meetups/meetups.module#MeetupsModule' },
+      { path: 'notificationviewlist', component: NotificationViewListComponent }

```

Comment:
Instead of such a big path name notificatonviewlist, we can use notification/view .